api/lib/domain/usecases/create-pro-organizations-with-tags-and-target-profiles.js

```diff
@@ -61,6 +63,17 @@ module.exports = async function createProOrganizationsWithTags({
     });
 
     await organizationTagRepository.batchCreate(organizationsTags, domainTransaction);
+
+    const organizationsTargetProfiles = createdOrganizations.flatMap(({ id, externalId }) => {
+      return organizationsData
+        .get(externalId)
```

Comment:
**Remarque** : D'après le ticket JIRA, l'externalId ne doit plus être obligatoire, pourtant on en dépend encore ici. On doit pouvoir se baser sur autre chose désormais

Reply:
C'etait un nice to have si c'est simple